### DISCUSSION TITLE
Enable Encrypt website with Let's Encrypt and hanging

### DISCUSSION THREAD

**Discussion by coldwinds** [0xJacky/nginx-ui#151] [Q&A] [answered]
Hello, 

When I Enable the SSL for my site, I got these error:

Nginx Configuration Parse Error
2023/08/07 02:49:46 [warn] 374#374: the "listen ... http2" directive is deprecated, use the "http2" directive instead in /etc/nginx/sites-enabled/test-site:43 nginx: [warn] the "listen ... http2" directive is deprecated, use the "http2" directive instead in /etc/nginx/sites-enabled/test-site:43 2023/08/07 02:49:46 [warn] 374#374: the "listen ... http2" directive is deprecated, use the "http2" directive instead in /etc/nginx/sites-enabled/test-site:44 nginx: [warn] the "listen ... http2" directive is deprecated, use the "http2" directive instead in /etc/nginx/sites-enabled/test-site:44 nginx: the configuration file /etc/nginx/nginx.conf syntax is ok nginx: configuration file /etc/nginx/nginx.conf test is successful

Do I need to take care of these errors?

**Comment by 0xJacky** [comment, marked as answer]
Please check out this post: https://forum.hestiacp.com/t/nginx-1-25-1-listen-http2-directive-is-deprecated/9816
